code/game/objects/items/tools/mining_tools.dm

```diff
@@ -149,7 +149,7 @@
 	else if(!powered)
 		to_chat(user, span_warning("[src] is turned off!"))
 	else
-		to_chat(user, span_warning("The plasma cutter has inadequate charge remaining! Replace or recharge the battery. <b>Charge Remaining: [cell.charge]/[cell.maxcharge]</b>"))
+		to_chat(user, span_warning("The plasma cutter has inadequate charge! Replace or recharge the battery. <b>Charge Amount: [cell.charge]/[cell.maxcharge]</b>"))
```

Comment:
Can you explain all these random remaining -> amount changes please

Reply:
Charge remaining looks odd when you gain charge upon hitting someone, and I wanted the message for battery level information to be the same across all PC messages. It also makes more sense considering the cell is self charging and cannot be removed.

Reply:
heavily debatable but i dont care enough